web/e2e/page-objects/flows/manageFlows.ts

```diff
@@ -216,14 +216,14 @@ export class ManageFlows extends AppPage {
    */
 
   async clickPaginationDropDown() {
-    return await element(by.id("mat-select-0")).click();
+    return await element(by.id("mat-select-arrow")).click();
   }
```

Comment:
mat-select-row is not an id, its a class. Please use by.css
Like earlier comment mat-select-row by itself returns 4 matches. Pls use by.css(".mat-select .mat-select-arrow")